crates/lovely-mac/Cargo.toml

```diff
@@ -13,3 +13,4 @@ lovely-core = { version ="0.5.0-beta7", path = "../lovely-core" }
 libc = "0.2.153"
 once_cell = "1.19.0"
 ctor = "0.2.7"
+log = "0.4.21"
```

Comment:
Unless I'm missing something you shouldn't need this - leftover from earlier development?